source/CardGames.py

```diff
@@ -278,134 +304,248 @@ def winnnerCheck(self, house, player):
-    blackjackOutcome = dealer.blackjackChecker(house, player_name)
-    if blackjackOutcome == 1:
-      print("The house has blackjack. You lose!")
-      betting_box.collect(blackjackOutcome)
-      print(f"You now have ${player_name.money}")
-    elif blackjackOutcome == 2:
-      print("Blackjack! You win!")
-      betting_box.collect(blackjackOutcome)
-      print(f"You now have ${player_name.money}")
-    elif blackjackOutcome == 3:
-      print("It's a push! You both have blackjack.")
-      betting_box.collect(blackjackOutcome)
-      print(f"You now have ${player_name.money}")
+      for letter in phrase:
+        print(letter, end="", flush=True), time.sleep(.01)
+
+      for dash in range(10):
+        print(" .",end="", flush=True), time.sleep(.1)
+      print()
+#####################################################################################################################
+def blackjackGame():
+  """blackjack Class initialization"""
+  bj = BlackJack()
+
+  """Get Player_name"""
+  player_name = input("Enter Username: ")
+
+  """player config settings and Deck, dealer initializations"""
+  for_money, deck_num, early_shuffle, player_name = bj.configSettings(player_name)
+  deck = Deck(deck_num)
+  deck.shuffle()
+  dealer = Dealer(deck)
+
+  """Player Class initializations"""
+  player_name = Player(player_name)
+  player_name.addMoney(500)
+  house = Player("house")
+
+  ####"""Game Loop"""####
+  betting = True
+  while betting:
+    hand_num = 1 # at the top of the loop, we are always looking at hand number 1
+
+    """This catches and disrupts the normal game loop if the player is in the middle of playing their split hands"""
+    while len(player_name.hands_lst) > 0:
+      current_hand = []
+      current_hand.append(player_name.hands_lst.pop(0))
+      player_name.hand = current_hand # rotating which hand to look at from the hands_lst
+      dealer.dealCards(1, [player_name])
+      print("Hand "+ str(hand_num) + " sum:", player_name.handSum())
+      player_name.showHand()
+
+      """Checks if the hand is blackjack. If not, then allow the player to complete the turn for that hand"""
+      if bj.blackjackDecider(house, player_name, betting_box, dealer, silent=True) == True:
+        player_name.completed_hands.append(player_name.hand)
+      else:
+        bj.playerTurn(player_name, dealer)
+        player_name.completed_hands.append(player_name.hand)
       
-    else:
-      """while the player doesn't bust, allow them to choose to hit or pass"""
-      bust = False
-      while not bust :
-        hit_stand = input("\nhit or stand? ")
-        if hit_stand == "hit":
-          dealer.dealCards(1,[player_name])
-          print("\nYour Hand: ", player_name.handSum())  
-          player_name.showHand()
-          if dealer.bustCheck(player_name) == True:
-            print("\nBust!")
-            bust = True
-        else:
+      """distinguish which hand is being looked at and slow down gameplay"""
```

Comment:
test

Reply:
Oh I see why you are having this issue. on line 377 there is this if statement:       

if hit_stand == "hit":
        dealer.dealCards(1,[player_name]) 

Because your input was 'h' it skipped to the else statement.